Kickstarter-iOS/ViewModels/AppDelegateViewModel.swift

```diff
@@ -282,10 +255,13 @@ AppDelegateViewModelOutputs {
       self.registerForRemoteNotifications = applicationLaunchedWithUser
     }
 
-    self.authorizeForRemoteNotifications = self.notificationAuthorizationStatusProperty.signal
-      .skipNil()
-      .filter { $0.isNotDetermined }
-      .ignoreValues()
+    self.authorizeForRemoteNotifications = Signal.merge(
+      applicationLaunchedWithUser,
+      self.notificationAuthorizationStatusProperty.signal
+        .skipNil()
+        .filter { $0 == .notDetermined }
+        .ignoreValues()
+    )
```

Comment:
As noted in the main comment, we want to request authorization both on app launch and if the result of `getNotificationSettings` is `.notDetermined`.